.rubocop.yml

```diff
@@ -2,6 +2,7 @@ require:
   - rubocop-rails
   - rubocop-rspec
   - rubocop-performance
+  - ./custom_cops/required_inverse_of_relations.rb
```

Comment:
I'm not sure if that works, but can we try to rename the folder: `./rubocop/cops/`. At first glance, `custom_cops` might be hard to recognize for people new in the world of Rails 🛤️

Reply:
Agree. Fixed in fba261b028c05b6a3fc2fa6e08450028ba607a89